helm-chart/templates/04-hub-deployment.yaml

```
@@ -0,0 +1,47 @@
+apiVersion: apps/v1
+kind: Deployment
```

```suggestion
kind: StatefulSet
```

Hub should be `StatefulSet`.

---

From what I saw its uses, it doesn't need to be StatefulSet. Workers can easily find it, as it keeps watching.

---

OK, let's keep it as `Deployment`.